lib/component.js

```diff
@@ -396,6 +419,29 @@ class Component extends WebComponent {
     return state;
   }
 
+  _updateAttrs(attr) {
```

Comment:
a brief comment description of this method would be useful (since its name sounds like it's performing multiple updates), also noting the side effect of updating `this.attrs` rather than returning anything